adopters.geojson

```diff
@@ -31,7 +31,6 @@
     {"type": "Feature", "properties": {"institution": "Technical University of Applied Sciences Lübeck"}, "geometry": {"type": "Point", "coordinates": [10.7002646, 53.837048]}},
     {"type": "Feature", "properties": {"institution": "Hochschule Bochum, Campus Velbert/Heiligenhaus"}, "geometry": {"type": "Point", "coordinates": [6.96794, 51.32762]}},
     {"type": "Feature", "properties": {"institution": "Hochschule Bochum"}, "geometry": {"type": "Point", "coordinates": [7.27248, 51.44711]}},
-    {"type": "Feature", "properties": {"institution": "INSA Lyon"}, "geometry": {"type": "Point", "coordinates": [4.8759893530497695, 45.7830905]}},
```

Comment:
You probably did not want to remove this institution, did you?

Reply:
Right. and I do not remember I did. so It might be a mistake. please see if you can discard that part of the patch.